tests/system/README.md

```diff
@@ -4,7 +4,25 @@ Wazuh - System quality assurance automation templates
 
 ## Setting up a test environment
 
-To run these tests we need to use a **Linux** machine and install the following tools:
+To run these tests it will be required to use a **Linux** machine.
```

Comment:
"A linux machine will be required"

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4536/commits/b30b1e13cf55de1b3faa64fea6350554f9914d8b